src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -29,18 +30,22 @@ public AnnotationView()
             Resources.MergedDictionaries.Add(SharedDictionaryManager.PortsDictionary);
 
             InitializeComponent();
-            Loaded += AnnotationView_Loaded;           
-            this.GroupTextBlock.MouseLeftButtonDown += UIElement_OnMouseLeftButtonDown;   
+            Loaded += AnnotationView_Loaded;                      
             this.GroupTextBlock.SizeChanged +=GroupTextBlock_SizeChanged;          
         }
      
         private void AnnotationView_Loaded(object sender, RoutedEventArgs e)
         {
             ViewModel = this.DataContext as AnnotationViewModel;
-            //Set the height of Textblock based on the content.
-            if (ViewModel != null && !ViewModel.AnnotationModel.loadFromXML)
+            if (ViewModel != null)
             {
-                ViewModel.AnnotationModel.TextBlockHeight = this.GroupTextBlock.ActualHeight;
+                //Set the event when Annotation view is loaded.
+                ViewModel.WorkspaceViewModel.Model.SetGetModelsForGrouping(ViewModel.AnnotationModel);
```

Comment:
So, does this feature not work in headless Dynamo?  It's pretty odd to be setting up `Model` layer event handling from the `View` layer.